gwt-ol3-client/src/main/java/ol/style/IconOptions.java

```diff
@@ -91,5 +91,11 @@
      */
     @JsProperty
     public native void setSrc(String imageUri);
+    
+    /**
+     * @param anchor set Anchor. Default value is [0.5, 0.5] (icon center).
+     */
+    @JsProperty
+    public native void setAnchor(Double[] anchor);
```

Comment:
No this is not a coordinate in the strict sense the image of the Icon has a grid 0,0 is top left and 1,1 is bottom right if noting is done then the icon is centered around the center of the icon. If the icon has some visible point then the standard positioning is not enough. This is not a Coordinate in the GIS sense but a positioning See OL reference: https://openlayers.org/en/latest/apidoc/ol.style.Icon.html

Reply:
@ikzent I know the API docs 😉  . It's not a coordinate in the GIS sense but it is a coordinate. Using a type facilitates the use of the API and  restricts invalid user inputs (Coordinate vs. Double[]). Coordinate type best fits in this case. But I can understand your concerns because the user could misunderstand the meaning. Therefore I think you could leave double array for the moment. But you should change it to primitive double.